bootstrap/bootstrap.go

```diff
@@ -227,6 +228,7 @@ func (b *Bootstrap) executeHook(ctx context.Context, hookCfg HookConfig) error {
 		"hook.name":    hookCfg.Name,
 		"hook.command": hookCfg.Path,
 	})
+	span.AddAttributes(hookCfg.SpanAttributes)
```

Comment:
Does this handle `hookCfg.SpanAttributes` being `nil`?

Reply:
Yep - both implementations of this method iterate over the map, which just does nothing on a nil map. I had to double check this, i hate nil maps so much.